HW03/matsvei_basiayeu/spec/matsvei_basiayeu_hw_03_t_04_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative 'spec_helper'
+require './matsvei_basiayeu_hw_03_t_04'
+
+RSpec.describe Homework3 do
+  describe 'task4' do
+    let(:obj) { described_class.new }
+
+    describe 'count letters and digits in the string' do
+      context 'when word consists digits and letters' do
+        it 'returns right number of letters and digits' do
+          expect(obj.task4('1245qwertyuiop')).to eq({ digits: 4, letters: 10 })
+        end
+      end
+
+      context 'when string is empty' do
+        it 'returns number of letters=0 and digits=0' do
```

Comment:
better rephrase like this: returns number of letters is 0 and digits is 0. it's a bad practice to use mathematical operators in spec description